cumulus_etl/common.py

```diff
@@ -318,6 +319,13 @@ def print_header(name: str | None = None) -> None:
     if name:
         print(name)
 
+def get_transient_progress()-> (progress.Progress):
```

Comment:
nit: this might more naturally fit in `cli_utils.py` (which is where `make_progress_bar()` lives). I'm trying to _roughly_ group misc functions. (Maybe even convert this into a `transient=False` kwarg on that method?)

How do you feel about whether a `TimeElapsed` column fits here? Helpful or noisy?

Also, why the parens on the return type hint? Is that something your IDE or `ruff` encourages?

Reply:
oh at one point i was passing back a tuple - i'll remove those.

TimeElapsed is a fine addition.